homeassistant/components/homematicip_cloud/binary_sensor.py

```diff
@@ -43,14 +43,25 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_LOW_BATTERY = "low_battery"
+ATTR_MOISTUREDETECTED = "moisture detected"
```

Comment:
State attribute keys should be lowercase snakecase.

Please separate the words in the constant with underscore.

```py
ATTR_MOISTURE_DETECTED = "moisture_detected"
```

Reply:
done

Reply:
Can we change the strings to snakecase too? It'll be a breaking change but as some strings are already snakecase and that is the standard, I think it would be for the better.

Reply:
Nevermind, I see the second commit now.